### PR TITLE
Add WPP tracing

### PR DESCRIPTION
**Pull request by aserdean**
- Add wpp tracing

- Demote a few log messages

- Document the different type of logging facilities

### CODE REVIEW

**Review by petrutlucian94 — COMMENTED** [review]
Nit: the title is incomplete, you can rename it to "Add driver WPP tracing" and then on the following line you may have something like "WPP reference:"

File: driver/userspace.c [hunk 1]
```diff
@@ -762,7 +762,7 @@ WnbdParseUserIOCTL(PWNBD_EXTENSION DeviceExtension,
             DeviceExtension, ReqCmd->ConnectionId, TRUE);
         if (!Device) {
             Status = STATUS_INVALID_HANDLE;
-            WNBD_LOG_WARN(
+            WNBD_LOG_INFO(
```
Comment (petrutlucian94) [inline driver/userspace.c:765]: I'd rather use a debug message since there can be hundreds of timed out requests.

File: README.md [hunk 2]
```diff
@@ -234,3 +234,56 @@ Number Friendly Name             Serial Number    HealthStatus         Operation
...
+
+To view and collect the log messages via `DbgPrint` you can use either a (debugger) [https://docs.microsoft.com/en-us/windows-hardware/drivers/debugger/]
+or, if you do not want to go through the process of attaching the debugger, you
+can simply use:
+
+(DebugView) [https://docs.microsoft.com/en-us/sysinternals/downloads/debugview]
+
+### WPP
+
+To collect messages using the `WPP` provider you can simply you can use the following snippet from an elevated command prompt:
```
Comment (petrutlucian94) [inline README.md:259]: This looks like a mistake: ``you can simply you can use``. Let's just go with ``you can use``.

**Comment by petrutlucian94** [comment]
Looks great, just a few nits.

File: README.md (hunk 2 above)
Comment (aserdean) [reply README.md:259]: Ooops.

**Pull request merged**